driver-core/src/main/java/com/datastax/driver/core/KeyspaceMetadata.java

```diff
@@ -262,21 +286,61 @@ public String exportAsString() {
 
         sb.append(asCQLQuery()).append('\n');
 
-        for (UserType udt : userTypes.values())
+        // include types, tables, views, functions and aggregates, each ordered by name, with one small exception
+        // being that user types are ordered topologically and then by name within same level.
+        for (UserType udt : getSortedUserTypes())
```

Comment:
If we re-sort when exporting as CQL, do we still need the changes from the first commit? Having a consistent order matters mostly for `exportAsString`, for a programmatic traversal I don't think it's a big deal if it differs across VMs.